net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -46,7 +46,7 @@ mwan3_rtmon_ipv4()
 		idx=$((idx+1))
 		tid=$idx
 		[ "$(uci get mwan3.@interface[$((idx-1))].family)" = "ipv4" ] && {
-			tbl=$($IP4 route list table $tid)
+			tbl=$($IP4 route list table $tid 2>/dev/null)
```

Comment:
What's the reasoning here? I've found mwan3 already has a tendency to send too many errors to /dev/null, which can make debugging difficult. When does ip route list output to stderr? Should it really be suppressed here?

Reply:
You're right that's hard to debug, but I always get the following output `dump terminated` coming from that line, if i do an `mwan3 restart` on the command line and my wwan interface is not connected. Maybe we have to consider if we don't write all output to a log. At least the one from stderr.

Reply:
Any idea why you are getting the `dump terminated` message? Sounds like something else is wrong, and it would be better to fix that than to mask the error here.

Reply:
This is because if the table is empty this will output.
On my setup it's right.
The interface has not yet been get connected  by the netifd, so if you restart mwan3 with `mwan3 restart' the table will be empty.

Reply:
The problem is also that if you start this on the cmd you get this message every interval of rtmon! That is boring